### PR TITLE
init holesky config for deploy use

### PR DESCRIPTION
**Pull request by MarvelFisher**

### CODE REVIEW

File: ops/L2-genesis/deploy-config/holesky-deploy-config.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "finalSystemOwner": "0x48442fdDd92F1000861c7A26cdb5c3a73FFF294d",
+  "proxyAdminOwner": "0x48442fdDd92F1000861c7A26cdb5c3a73FFF294d",
+
+  "l1FeeVaultRecipient": "0x484426a2F7e0469d70a51cd07cCB7793d58e9008",
+  "sequencerFeeVaultRecipient": "0x484426a2F7e0469d70a51cd07cCB7793d58e9008",
+  "l2BridgeFeeVaultRecipient": "0x484426a2F7e0469d70a51cd07cCB7793d58e9008",
+  "fundDevAccounts": false,
+
+  "l1StartingBlockTag": "earliest",
+  "maxTxPerBlock": 1000,
```
Comment (FletcherMan) [inline ops/L2-genesis/deploy-config/holesky-deploy-config.json:14]: should be 100

File: ops/L2-genesis/deploy-config/holesky-deploy-config.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  "gasPriceOracleOverhead": 2500,
+  "gasPriceOracleScalar": 1000000000,
+  "gasPriceOracleOwner": "0x48442fdDd92F1000861c7A26cdb5c3a73FFF294d",
+
+  "govProposalInterval": 1000,
+  "govBatchBlockInterval": 20,
```
Comment (FletcherMan) [inline ops/L2-genesis/deploy-config/holesky-deploy-config.json:21]: `govBatchBlockInterval` should be 600

File: ops/L2-genesis/deploy-config/holesky-deploy-config.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  "govBatchMaxBytes": 124928,
+  "govBatchTimeout": 600,
```
Comment (FletcherMan) [inline ops/L2-genesis/deploy-config/holesky-deploy-config.json:23]: `govBatchTimeout` should be 1200

File: ops/L2-genesis/deploy-config/holesky-deploy-config.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  "govRollupEpoch": 100,
```
Comment (FletcherMan) [inline ops/L2-genesis/deploy-config/holesky-deploy-config.json:24]: `govRollupEpoch`  should be 600

**Review by Kukoomomo — APPROVED** [review]

**Pull request merged**